### DISCUSSION TITLE
Good moment to refactor and get back ChatGPT cookie-based code for GPT-4 access?

### DISCUSSION THREAD

**Discussion by antonme** [danny-avila/LibreChat#71] [Feature Requests & Suggestions]
It is now possible to use GPT-4 for ChatGPT Plus users via ChatGPT interface. I wonder if it is possible to use pre-normal API mechanism with a cookie access to ChatGPT to add it to accessible models as well.

<img width="726" alt="Screenshot 2023-03-14 at 23 03 53" src="https://user-images.githubusercontent.com/1531009/225123778-fbd96b97-a283-4c4f-9254-f6066cd6d9b0.png">

**Comment by danny-avila** [comment]
This is better to suited to discuss in the api project this one uses as it hasn't been brought up yet: https://github.com/waylaidwanderer/node-chatgpt-api
